addons/revolutionaries/CfgVehicles_I_Russian.hpp

```diff
@@ -71,6 +71,75 @@ class TACU_Revolutionaries_U_I_Russian_Marksman: TACU_Revolutionaries_U_I_Russia
     editorPreview = QPATHTOF(ui\Revolutionaries_U_I_Russian_Marksman.jpg);
 };
 
+class TACU_Revolutionaries_U_I_Russian_Pistoleer01: TACU_Revolutionaries_U_I_Russian_Rifleman01 {
+    displayName = "Pistoleer (Garch)";
+    uniformClass = "CUP_U_C_racketeer_03";
+    backpack = "";
+    weapons[] = {"hgun_Rook40_F", "Throw", "Put"};
+    respawnWeapons[] = {"hgun_Rook40_F", "Throw", "Put"};
+    magazines[] = {mag_6("16Rnd_9x21_Mag")};
+    respawnMagazines[] = {mag_6("16Rnd_9x21_Mag")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_I_Russian_Pistoleer02: TACU_Revolutionaries_U_I_Russian_Rifleman01 {
+    displayName = "Pistoleer (FNX)";
+    uniformClass = "CUP_U_C_racketeer_04";
+    backpack = "";
+    weapons[] = {"hgun_Pistol_heavy_01_F", "Throw", "Put"};
+    respawnWeapons[] = {"hgun_Pistol_heavy_01_F", "Throw", "Put"};
+    magazines[] = {mag_6("11Rnd_45ACP_Mag")};
+    respawnMagazines[] = {mag_6("11Rnd_45ACP_Mag")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
+};
+
+class TACU_Revolutionaries_U_I_Russian_Head: TACU_Revolutionaries_U_I_Russian_Rifleman01 {
+    displayName = "Head";
+    uniformClass = "CUP_U_C_Profiteer_04";
+    linkedItems[] = {DEFAULT_ITEMS, "V_LegStrapBag_black_F", "CUP_H_C_Beret_04"};
+    respawnLinkedItems[] = {DEFAULT_ITEMS, "V_LegStrapBag_black_F", "CUP_H_C_Beret_04"};
+    backpack = "";
+    weapons[] = {"hgun_P07_F", "Throw", "Put"};
+    respawnWeapons[] = {"hgun_P07_F", "Throw", "Put"};
+    magazines[] = {mag_6("16Rnd_9x21_Mag")};
+    respawnMagazines[] = {mag_6("16Rnd_9x21_Mag")};
+    //editorPreview = QPATHTOF(ui\Revolutionaries_U_O_Russian_Autorifleman.jpg);
```

Comment:
What's the point of `headgearList[]` if it only contains a single item that will always be there (`weight = 1`)?

Reply:
Perhaps it's my misunderstanding of how exactly it works then.
It was my belief that since this unit inherits from Rifleman01, where it's set as:
```
headgearList[] = {
    "", 1
};
```
then the headgearList for this unit should be added to prevent the beret from being overwritten?

Reply:
Ah, if that's the case, it's good.